apps/hasura/config.yaml

```
@@ -1,6 +1,7 @@
 version: 3
 endpoint: http://localhost:8080
 metadata_directory: metadata
+# admin_secret: lfg  Local 
```

yamllint

[error] trailing spaces

(trailing-spaces)